features/ouders.feature

```diff
@@ -6,7 +6,7 @@ Functionaliteit: Ouders van een persoon raadplegen
   Dit bevat gegevens over de familierechtelijke betrekking plus enkele identificerende eigenschappen van de ouder.
 
 @gba
-Rule: "ouder1" is de ouder in categorie 2 en "ouder2" is de ouder in categorie 3
+Rule: "1" is de ouder in categorie 2 en "2" is de ouder in categorie 3
```

Comment:
deze zin is nu wat cryptisch geworden. Misschien zoiets:
Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie 3 heeft ouderAanduiding "2"

Reply:
Overgenomen zoals voorgesteld